service/frontend/wrappers/clusterredirection/policy.go

```diff
@@ -264,19 +269,32 @@ func (policy *selectedOrAllAPIsForwardingRedirectionPolicy) withRedirect(
 	call func(string) error,
 ) error {
 	targetDC, enableDomainNotActiveForwarding := policy.getTargetClusterAndIsDomainNotActiveAutoForwarding(ctx, domainEntry, workflowExecution, actClSelPolicyForNewWF, apiName, requestedConsistencyLevel)
+	domainName := domainEntry.GetInfo().Name
 
-	policy.logger.Debugf("Calling API %q on target cluster:%q for domain:%q", apiName, targetDC, domainEntry.GetInfo().Name)
+	policy.logger.Debugf("Calling API %q on target cluster:%q for domain:%q", apiName, targetDC, domainName)
```

Comment:
nit, generally prefer structured logging over string interpolated logging as it's easier to search with. Debug is probably nbd however.